.gitignore

```diff
@@ -6,3 +6,4 @@
 /releng/__pycache__/
 /releng/modules/frida-gadget-ios/*.dylib
 /releng/modules/frida-gadget-ios/node_modules
+/g-ir-cpp-*.c
```

Comment:
Explanation here: if `g-ir-scanner` runs into an error, it will dump this `.c` file into the root of the repository